python/pyspark/pandas/tests/test_categorical.py

```diff
@@ -560,6 +560,38 @@ def test_unstack(self):
         self.assert_eq(psdf.a.unstack().sort_index(), pdf.a.unstack().sort_index())
         self.assert_eq(psdf.b.unstack().sort_index(), pdf.b.unstack().sort_index())
 
+    def test_rename_categories(self):
+        pdf, psdf = self.df_pair
+
+        pser = pdf.b
+        psser = psdf.b
+
+        self.assert_eq(
+            pser.cat.rename_categories([0, 1, 3, 2]), psser.cat.rename_categories([0, 1, 3, 2])
+        )
+        self.assert_eq(
+            pser.cat.rename_categories({"a": "A", "c": "C"}),
+            psser.cat.rename_categories({"a": "A", "c": "C"}),
+        )
+        self.assert_eq(
+            pser.cat.rename_categories(lambda x: x.upper()),
+            psser.cat.rename_categories(lambda x: x.upper()),
+        )
+        pser.cat.rename_categories({"a": "A", "c": "C"}, inplace=True)
+        psser.cat.rename_categories({"a": "A", "c": "C"}, inplace=True)
+        self.assert_eq(pser, psser)
+        pser.cat.rename_categories(lambda x: x.upper(), inplace=True)
+        psser.cat.rename_categories(lambda x: x.upper(), inplace=True)
+        self.assert_eq(pser, psser)
+        pser.cat.rename_categories(lambda x: x.upper(), inplace=True)
+        psser.cat.rename_categories(lambda x: x.upper(), inplace=True)
+        self.assert_eq(pser, psser)
```

Comment:
Why is this doing the same thing as the above?

Reply:
Oh good catch, thanks!